src/arm/radxa_rock_3c.c

```diff
@@ -49,6 +49,15 @@ mraa_radxa_rock_3c()
         return NULL;
     }
 
+    if (mraa_file_contains("/proc/device-tree/model", PLATFORM_NAME_RADXA_ROCK_3C)) {
+        b->platform_name = PLATFORM_NAME_RADXA_ROCK_3C;
+    } else if (mraa_file_contains("/proc/device-tree/model", PLATFORM_NAME_RADXA_ZERO3)) {
+        b->platform_name = PLATFORM_NAME_RADXA_ZERO3;
+    } else {
+        printf("An unknown product detected. Fail early...\n");
+        exit(-1);
```

Comment:
```suggestion
        return NULL;
```
你区区一个运行库凭什么把我主程序都给干掉了？